app/routes/pull-requests.js

```diff
@@ -18,13 +33,25 @@ export default class PullRequestsRoute extends Route {
   setupController(controller, model) {
     super.setupController(controller, model);
 
-    controller.mergedPRs = filterMerged(model);
-    controller.newPRs = filterNew(model);
+    controller.mergedPRs = filterMerged({
+      pullRequests: model,
+      mergedSince: this.mergedSince,
+    });
+
+    controller.newPRs = filterNew({
+      pullRequests: model,
+      mergedSince: this.mergedSince,
+    });
   }
 
   async fetchPRs(organizations) {
+    const createdSince = moment(this.mergedSince).subtract(2, 'weeks').format('YYYY-MM-DD');
```

Comment:
We may be able to extract a function and save it in `pull-request` utility. The important thing to note is that `mergedSince` and `createdSince` are both date strings, in the format of `YYYY-MM-DD`.